### PR TITLE
version 2.0.0

### PR DESCRIPTION
**Pull request by robert-lytton**
PDF documentation needs tidying ("\exception" not being handled)
xdoc also incorrectly reporting error due to the string "error" being present in the text.

### CODE REVIEW

**Comment by xross** [comment]
Ideally magic numbers such as:

asm volatile("setc res[%0], 0x1" :: "r" (tmr));

should come from xs**_user.h etc. E.g. the above should use XS1_RES_TYPE_TIMER

**Comment by robert-lytton** [comment]
It means adding them in C and passing them in as registers into the inline asm.
BUT it is easier to read/verify so will make the changes.
Thank you.

File: README.rst [hunk 1]
```diff
@@ -16,7 +16,8 @@ Features
    with XC channels and the ability to write custom channel protocols
  * Support for ports and clock blocks
  * Support for timers
- * Support for events so that XC 'select' functionality can be implemented
+ * Support for select events so that XC 'select' functionality can be implemented
```
Comment (samchesney) [inline README.rst]: Minor, but I think it should be styled with a lowercase 'x', as in xC.

File: lib_xcore_c/api/xcore_c_chanend.h [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+ *
+ *  \exception  ET_ILLEGAL_RESOURCE   not a valid chan-end.
+ *  \exception  ET_RESOURCE_DEP       another core is actively using the chan-end.
+ */
+inline xcore_c_error_t chanend_disable_trigger(chanend c)
+{
+  RETURN_EXCEPTION_OR_ERROR( _resource_disable_trigger(c) );
+}
+
+#endif // __XC__
```
Comment (samchesney) [inline lib_xcore_c/api/xcore_c_chanend.h]: Might avoid confusion for the "casual reader" (e.g. people that haven't studied the file from the top) if this comment was ``! __XC__``?

**Comment by samchesney** [comment]
The changes to the API all look sensible to me 👍

**Comment by robert-lytton** [comment]
done

**Comment by robert-lytton** [comment]
Ready to merge

**Pull request merged**